rate-limit-redis/README.md

```diff
@@ -0,0 +1,68 @@
+# Redis Rate Limiter Middleware for Fiber
```

Comment:
pls follow the https://github.com/gofiber/recipes/blob/master/.github/CONTRIBUTING.md steps